test/recovery/truncated-log.c

```diff
@@ -193,9 +189,9 @@ fill_db(void)
 	max_key = min_key * 2;
 	first = true;
 	for (i = 0; i < max_key; ++i) {
-		snprintf(k, sizeof(k), "key%03d", (int)i);
-		snprintf(v, sizeof(v), "value%0*d",
-		    (int)(V_SIZE - strlen("value")), (int)i);
+		testutil_check(__wt_snprintf(k, sizeof(k), "key%03d", (int)i));
+		testutil_check(__wt_snprintf(v, sizeof(v), "value%0*d",
+		    (int)(V_SIZE - (strlen("value") + 1)), (int)i));
```

Comment:
There's an off-by-one here, we weren't taking the buffer's nul-termination into account.